table.go

```diff
@@ -231,11 +238,36 @@ func (t *table) Print() {
 	t.calculateWidths()
 
 	t.printHeader(format)
+	if t.HeaderSeparatorRune != 0 {
+		t.printHeaderSeparator(format)
+	}
 	for _, row := range t.rows {
 		t.printRow(format, row)
 	}
 }
 
+func (t *table) printHeaderSeparator(format string) {
+	separators := make([]string, len(t.header))
+	separatorRuneWidth := t.Width(string([]rune{t.HeaderSeparatorRune}))
+	for index, headerName := range t.header {
+		headerRuneCount := t.Width(headerName)
```

Comment:
This is technically not a rune count but rather a monospace-width count (i.e., how many columns in a monospace font are taken by this string)

Reply:
Yeah, I initially called `utf8.RuneCountInString`. Forgot to rename :D

Will fix